Update README.md Docker Commands and default udp port (#55)
foxssake/noray: README.md

```
docker build . -t noray

> docker run -p 8890:8890 -p 8890:8890/udp -p 8091:8091 --env-file=.env -t noray

Reviewer: Based on default config, noray listens on 8890 TCP for commands, 8891 UDP for registrar, and 8891 HTTP for exposing metrics.

```suggestion
docker run -p 8890:8890 -p 8891:8891/udp -p 8891:8891 --env-file=.env -t noray
```

```

Or run prebuilt docker:
```

> docker run -p 8890:8890 -p 8890:8890/udp -p 8091:8891 --env-file=.env -t ghcr.io/foxssake/noray:main

Reviewer: Same as above.

```suggestion
docker run -p 8890:8890 -p 8891:8891/udp -p 8891:8891 --env-file=.env -t ghcr.io/foxssake/noray:main
```

```

> _Please Note: This configuration will only allow NAT Punching but no port forwarding. Make sure to add `-p 49152-51200:49152-51200/udp` to allow all default udp relay ports. Beware! This will increase the deploy time severely!_

Reviewer: I'd prefer to call it relaying, just to stay consistent with the existing docs.

```suggestion
_Please Note: This configuration will only allow NAT punchthrough but no relaying. Make sure to add `-p 49152-51200:49152-51200/udp` to allow all default udp relay ports. Beware! This will increase the deploy time severely!_
```
